pkg/apis/clusteroperator/types.go

```diff
@@ -81,6 +81,14 @@ type AWSClusterSpec struct {
 	// credentials
 	AccountSecret corev1.LocalObjectReference
 
+	// SSHSecret refers to a secret that contains the ssh private key to access
+	// EC2 instances for this cluster
+	SSHSecret corev1.LocalObjectReference
+
+	// SSHSecretName is the name of the secret to use for SSH access to EC2
+	// instances in this cluster
+	SSHSecretName string
```

Comment:
Why do we need a SSHSecretName as well as the SSHSecret object ref? Looks like it may be unused.

Reply:
The secret name is needed so you can use a known secret name like "libra" (if not we can create one for you)

The secret itself is for the ssh private key that we will use when ssh'ing into created instances.